examples/30_extended/run_setup_tutorial.py

```diff
@@ -52,22 +57,39 @@
 # we will create a fairly complex model, with many preprocessing components and
 # many potential hyperparameters. Of course, the model can be as complex and as
 # easy as you want it to be
-model_original = sklearn.pipeline.make_pipeline(
-    sklearn.impute.SimpleImputer(), sklearn.ensemble.RandomForestClassifier()
-)
 
+from sklearn.ensemble import HistGradientBoostingClassifier
```

Comment:
What's the reason we are using the experimental `HistGradientBoostingClassifier` instead of a `RandomForestClassifier`?

Reply:
Addressed in #967.